app/models/crop.rb

```diff
@@ -41,6 +41,17 @@ def main_image_path
     if pictures.present?
       pictures.first.attachment.url
     else
+      # WARNING! MVC VIOLATION AHEAD!!!! =======================================
```

Comment:
This guy knows whats up (first response on google for "Rails Presenter Object") http://nithinbekal.com/posts/rails-presenters/